meinberlin/apps/cms/models/pages.py

```diff
@@ -12,7 +12,7 @@ class SimplePage(Page):
     body = fields.RichTextField(blank=True)
 
     content_panels = [
-        panels.FieldPanel("title"),
+        panels.TitleFieldPanel("title"),
```

Comment:
@goapunk @philli-m this change is necessary if the new pages get the slug from their title. Is that the case? When i create a new page in wagtail they are just assigned a number.

Reply:
@m4ra I think on most page types we auto generate the slug from the page title but it can be edited in the 'promote' panel in wagtail page editor. What page type did you create that was assigned a number?

Reply:
@m4ra yes, we want the slug from the title here, so using `TitleFieldPanel` is correct.